### PR TITLE
Edge case where 'args' is a string passed to display_args

### PR DESCRIPTION
**Pull request by suddenrushofsushi**
Why:
In rare cases the UI at /sidekiq/morgue will throw a 500 error with: undefined method `map' for "value":String; especially when third party tools can submit to a queue.

What:
This PR handles the edge case where a job's arguments are passed through to the display helper as a string, instead of an array. It also adds some minimal tests for the display_args helper.

### CODE REVIEW

File: lib/sidekiq/web_helpers.rb [hunk 1]
```diff
@@ -166,6 +166,7 @@ def truncate(text, truncate_after_chars = 2000)
     end
 
     def display_args(args, truncate_after_chars = 2000)
+      args = [args] unless args.respond_to?(:map)
```
Comment (davydovanton) [inline lib/sidekiq/web_helpers.rb]: may be is better to use `Array()` instead conditional? I.e. `Array(args).map {}`

Comment (suddenrushofsushi) [inline lib/sidekiq/web_helpers.rb]: Yes, I agree. I'll update that, thank you.

**Comment by mperham** [comment]
Args must be an array. That's the job payload spec. Garbage in, garbage out.

**Comment by suddenrushofsushi** [comment]
Alright, thanks. Closing.
